test/metabase/driver/sql_jdbc/sync/describe_table_test.clj

```diff
@@ -152,6 +153,16 @@
                #'sql-jdbc.describe-table/describe-json-xform
                #'sql-jdbc.describe-table/describe-json-rf [json-map]))))))
 
+(deftest get-table-pks-test
+  ;; FIXME: this should works for all sql drivers
+  (mt/test-drivers #{:mysql :postgres}
```

Comment:
just in case, lets make sure this works for all json column drivers
```suggestion
  (mt/test-drivers (mt/normal-drivers-with-feature :nested-field-columns)
```